### PR TITLE
Release v0.7

### PR DESCRIPTION
**Pull request by rn**
It has been quite a while.

Note, once merged, the repository is locked till the actual release

DqqYhr0WkAA7ykR

### CODE REVIEW

**Comment by rn** [comment]
The kernel modules test failed due to a mis-match in gcc version (the latest alpine base as a newer version of gcc). Updating the kernel build and rebuilding the kernels

File: AUTHORS [hunk 1]
```diff
@@ -80,6 +81,7 @@ Mathieu Pasquet <mathieu.pasquet@alterway.fr>
 Matt Bajor <matt.bajor@workday.com>
 Matt Bentley <matt.bentley@docker.com>
 Matt Johnson <matjohn2@cisco.com>
+mcy <james@mcy.email>
```
Comment (justincormack) [inline AUTHORS]: this is a dup of above

**Comment by justincormack** [comment]
one minor nit but lgtm

**Comment by rn** [comment]
Force pushed an update. the kernel compile is not quite finished yet but might win the race with the CI. If not, will re-kick CI later

**Comment by justincormack** [comment]
All green...

**Pull request merged**